lib/active_merchant/billing/gateways/worldpay_us.rb

```diff
@@ -151,7 +174,7 @@ def commit(action, post)
 
         raw = parse(ssl_post(live_url, post.to_query))
 
-        succeeded = success_from(raw['transresult'])
+        succeeded = success_from(raw['result'])
```

Comment:
@mrezentes this seems like a breaking change - could it have ever worked using `transresult`?